pkg/scheduler/core/generic_scheduler.go

```diff
@@ -182,14 +182,16 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 			FailedPredicates: failedPredicateMap,
 		}
 	}
-	metrics.SchedulingAlgorithmPredicateEvaluationDuration.Observe(metrics.SinceInMicroseconds(startPredicateEvalTime))
+	metrics.SchedulingAlgorithmPredicateEvaluationDuration.Observe(metrics.SinceInSeconds(startPredicateEvalTime))
```

Comment:
Are you saying replacing "microseconds" with "seconds" is a common pattern? If yes, can you show that where it's documented?

Reply:
Correct.
From the KEP [kubernetes-metrics-overhaul](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/0031-kubernetes-metrics-overhaul.md), Kubernetes metrics should follow Prometheus best practices.
The `seconds` is the [suggested base unit](https://prometheus.io/docs/practices/naming/#base-units) for time type metrics. And `seconds` already been widely used in other Kubernetes metrics.
So we should change these metrics to let them more consistent with others.